networkit/test/test_distance.py

```diff
@@ -99,53 +99,94 @@ def eucledianDist(u):
 		testMesh(9, 18)
 		testMesh(7, 1)
 	
-	def testDiameter(self):
+	def testAdamicAdarDistance(self):
+		for g in self.genERGraphs():
```

Comment:
Testing on one graph is enough for Python tests, the same applies for the other tests below.